newlib/libc/tinystdio/conv_flt.c

```diff
@@ -245,7 +245,29 @@ conv_flt (FLT_STREAM *stream, FLT_CONTEXT *context, width_t width, void *addr, u
                 return 0;
 	    }
         }
-	break;
+        if(CHECK_WIDTH() && (i = scanf_getc (stream, context)) == '(') 
```

Comment:
It looks like this will also (incorrectly) allow for `INF(abcd)`? To avoid this, I think you can check to see if `p == pstr_an + 3` before the `CHECK_WIDTH()` invocation.

Reply:
Done.